src/app/home-page/home-page.module.ts

```diff
@@ -3,9 +3,11 @@ import { CommonModule } from '@angular/common';
 
 import { HomePageRoutingModule } from './home-page-routing.module';
 import { HomePageComponent } from './components/home-page';
+import { CarouselComponent } from '../shared/components/carousel/carousel.component';
+
 
 @NgModule({
-  declarations: [HomePageComponent],
+  declarations: [HomePageComponent, CarouselComponent],
```

Comment:
CarouselComponent is a shared component. So it should be added to the SharedModule declaration. And HomePageModule should import SharedModule only.

We talked about it yesterday